config/locales/vi.yml

```diff
@@ -41,4 +44,12 @@ vi:
       not_found_user: "Không tìm thấy người dùng"
     edit:
       update_profile: "Cập nhật thông tin cá nhân"
+  user_mailer:
```

Comment:
File này thiếu phải k, hay là file en dư?
index:
      all_users: "All Users"
      not_found_user: "User not found !"
    delete: "DELETE ?"